homeassistant/components/decora_wifi/config_flow.py

```diff
@@ -0,0 +1,113 @@
+"""Config flow for myLeviton decora_wifi."""
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_PASSWORD, CONF_USERNAME
+
+from .common import CommFailed, DecoraWifiPlatform, LoginFailed
+from .const import CONF_TITLE, DOMAIN
+
+
+@config_entries.HANDLERS.register(DOMAIN)
+class DecoraWifiConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a Decora Wifi config flow."""
+
+    VERSION = 1
+
+    def __init__(self):
+        """Initialize the config flow."""
+        self.session = None
+        self.data = {}
+        self.entry = None
+
+    # Disable pylint error about number of arguments in step_user
+    # pylint: disable=arguments-differ
+    async def async_step_user(self, user_input=None, errors=None):
```

Comment:
We aren't allowed to pass errors between steps

Reply:
Corrected.